com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -1078,11 +1091,12 @@ void SetRenderPassAttachments(CommandBuffer cmd, ScriptableRenderPass renderPass
 
                 // When render pass doesn't call ConfigureTarget we assume it's expected to render to camera target
                 // which might be backbuffer or the framebuffer render textures.
+
                 if (!renderPass.overrideCameraTarget)
                 {
                     // Default render pass attachment for passes before main rendering is current active
                     // early return so we don't change current render target setup.
-                    if (renderPass.renderPassEvent < RenderPassEvent.BeforeRenderingOpaques)
+                    if (renderPass.renderPassEvent < RenderPassEvent.BeforeRenderingPrePasses)
```

Comment:
We need to check if this causes a behaviour change regarding clearing. I think @Verasl mentioned this PR caused a different in clear in boat attack. It could be caused due to this.